spec/features/admins/view_users_spec.rb

```diff
@@ -5,14 +5,20 @@
 
   scenario "Admin can see a list of all the users" do
     admin = FactoryGirl.create(:user, role: "admin")
-    user = FactoryGirl.create(:user)
+    next_page_user = FactoryGirl.create(:user, username: "nextpageuser")
+    users = Array.new
+    25.times { |u| users << FactoryGirl.create(:user) }
```

Comment:
Unused block argument - `u`. You can omit the argument if you don't care about it.